src/App.vue

```diff
@@ -20,34 +20,39 @@
           width="100"
         />
       </div>
-
-      <v-spacer></v-spacer>
```

Comment:
any reason to remove this?

Reply:
Actually spacer tag is removed to fix the position of multilingual dropdown